crates/sncast/tests/e2e/main_tests.rs

```diff
@@ -33,6 +32,8 @@ async fn test_happy_case_from_sncast_config() {
 
 #[tokio::test]
 async fn test_happy_case_from_cli_no_scarb() {
+    let tempdir = tempdir().expect("Unable to create temporary directory");
```

Comment:
this is kid of pointless, as call does not modify fs, and does not depend in it in any meaningful way (eg reading hashes/addresses etc)

Reply:
But since the test is being executed by default in `repo/crates/sncast/` there is very little chance there is a config file there, right? (it shouldn't be, and it won't ever be in repo).
To properly make sure about the assumption you've mentioned, we could rather assert for arguments that were used in cli, but I don't think this is necessary tbh